### PR TITLE
Removed Doorkeeper gem

### PR DESCRIPTION
**Pull request by gfedorenko**

### CODE REVIEW

File: app/controllers/private/api_tokens_controller.rb [hunk 1]
```diff
@@ -7,10 +7,8 @@ def index
       @tokens = current_user.api_tokens.user_requested
       @oauth_api_tokens = current_user.api_tokens.oauth_requested
 
-      ids = Doorkeeper::AccessToken
-        .where(id: @oauth_api_tokens.map(&:oauth_access_token_id))
-        .group(:application_id).select('max(id) as id')
-      @oauth_access_tokens = Doorkeeper::AccessToken.where(id: ids).includes(:application)
+      ids = APIToken.where(id: @oauth_api_tokens.map(&:oauth_access_token_id)).select('max(id) as id')
```
Comment (yivo) [inline app/controllers/private/api_tokens_controller.rb]: @gfedorenko we need to remove every oauth2 stuff. APIToken should be very very simple server2server token.

File: app/models/api_token.rb [hunk 2]
```diff
@@ -2,7 +2,7 @@ class APIToken < ActiveRecord::Base
   paranoid
 
   belongs_to :member
-  belongs_to :oauth_access_token, class_name: 'Doorkeeper::AccessToken', dependent: :destroy
+  belongs_to :oauth_access_token, dependent: :destroy
```
Comment (yivo) [inline app/models/api_token.rb]: @gfedorenko Don't forget to cleanup database schema.

**Review by yivo — CHANGES_REQUESTED** [review]

File: db/migrate/20171221131530_drop_doorkeeper_tables.rb [hunk 3]
```diff
@@ -0,0 +1,8 @@
+class DropDoorkeeperTables < ActiveRecord::Migration
+  def change
+    drop_table :oauth_applications
+    drop_table :oauth_access_grants
+    drop_table :oauth_access_tokens
+    remove_column :api_tokens, :oauth_access_token_id, :integer
```
Comment (mod) [inline db/migrate/20171221131530_drop_doorkeeper_tables.rb]: I don't think api_tokens relates to doorkeeper.

**Review by mod — CHANGES_REQUESTED** [review]
I am not comfortable to understand why they had doorkeeper in the first place

File: db/migrate/20171221131530_drop_doorkeeper_tables.rb (hunk 3 above)
Comment (yivo) [reply db/migrate/20171221131530_drop_doorkeeper_tables.rb]: @gfedorenko Can you please remove `:integer` from the list?

**Review by yivo — CHANGES_REQUESTED** [review]

**Review by yivo — APPROVED** [review]

**Pull request merged**